### DISCUSSION TITLE
Represent scores as sparklines

### DISCUSSION THREAD

**Discussion by ojanvafai** [iSoron/uhabits#280] [Feature Requests]
Not sure this is a good idea, but I could imagine it being nice to show a sparkline of the score insted of the circle so you could see the trend of your progress instead of just the current value. Basically the habit strength graph as a single line tiny icon.

**Comment by iSoron** [comment]
Thank you for the suggestion, @ojanvafai. I think this could work. Even if not replacing the circle, we could certainly show sparklines in other places. If anyone wants to implement a prototype, or even just prepare some mockups, I would be glad to consider this feature.
